scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -1667,3 +1670,459 @@ cdef FibonacciNode* remove_min(FibonacciHeap* heap) noexcept:
 #        print_node(heap.min_node)
 #    else:
 #        print("[empty heap]")
+
+######################################################################
+
+# Author: Tomer Sery  -- <tomersery28@gmail.com>
+# License: BSD 3-clause ("New BSD License"), (C) 2024
+
+def yen(
+    csgraph,
+    source,
+    sink,
+    K,
+    *,
+    directed=True,
+    return_predecessors=False,
+    unweighted=False,
+):
+    """
+    yen(csgraph, source, sink, K, *, directed=True, return_predecessors=False,
+        unweighted=False)
+
+    Yen's K-Shortest Paths algorithm on a directed or undirected graph.
+
+    .. versionadded:: 1.13.0
```

Comment:
```suggestion
    .. versionadded:: 1.14.0
```
It looks like this isn't going to make it into the next release branch.  It will need to be 1.14.

Reply:
Done